.github/workflows/copy_to_gitbook_repo.yml

```diff
@@ -0,0 +1,45 @@
+name: Copy file to repository synced with Gitbook
+
+on:
+  workflow_call:
+    inputs:
+      source_file:
+        required: true
+        type: string
+      destination_folder:
+        type: string
+      rename:
+        type: string
+      action_name:
+        type: string
+    secrets:
+      access_token:
+        required: true
+
+jobs:
+  copy_file:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Copy File to Another Repo
+        uses: nufi-official/copy_file_to_another_repo_action@main
+        env:
+          API_TOKEN_GITHUB: ${{ secrets.access_token }}
```

Comment:
Can we rename the secret to be more specific? Is it the token of the source repository?